concent_api/core/views.py

```diff
@@ -524,9 +639,19 @@ def validate_task_id(task_id):
         raise Http400("task_id cannot be longer than {} chars.".format(MESSAGE_TASK_ID_MAX_LENGTH))
 
 
-def store_message_and_message_status(golem_message_type: int, task_id: str, raw_golem_message: bytes, status = None, delivered: bool = False):
+def store_message_and_message_status(
+    golem_message_type:     int,
+    task_id,
+    raw_golem_message:      bytes,
+    provider_public_key:    str,
+    requestor_public_key:   str,
+    status                          = None,
```

Comment:
Wouldn't using `type` as the type here work?

Reply:
:+1: